templates/zerver/api/server-settings.md

```diff
@@ -0,0 +1,70 @@
+# Get global settings
+
+Fetch global settings for the Zulip server.
+
+`GET {{ api_url }}/v1/server_settings`
+
+## Usage examples
+
+<div class="code-section" markdown="1">
+<ul class="nav">
+<li data-language="python">Python</li>
+<li data-language="curl">curl</li>
+</ul>
+<div class="blocks">
+
+<div data-language="curl" markdown="1">
+
+```
+curl {{ api_url }}/v1/server_settings \
+    -u BOT_EMAIL_ADDRESS:BOT_API_KEY
+```
+
+</div>
+
+<div data-language="python" markdown="1">
+
+{generate_code_example(python)|/server_settings:get|example}
+
+</div>
+
+</div>
+
+</div>
+
+## Arguments
+
+{generate_api_arguments_table|zulip.yaml|/server_settings:get}
+
+## Response
+
+#### Return values
+
+* `authentication_methods`: object in which each key-value pair in the object
+  indicates if the authentication method in the key is enabled or not.
+* `zulip_version`: the version of Zulip running in the server.
+* `push_notifications_enabled`: whether mobile/push notifications are enabled
+  or not.
+* `email_auth_enabled`: setting for allowing users authenticate with an
+  email-password combination.
+* `require_email_format_usernames`: whether usernames should have an email
+  address format or not.
```

Comment:
We should mention that this is important for LDAP authentication with username+password, and link to our production docs on authentication methods (which I don't think cover LDAP in detail yet, but will eventually).